schemas/Protocol

```diff
@@ -71,52 +106,158 @@
             }
         },
         {
-            "@id": "skos:prefLabel",
+            "@id": "reproterms:order",
             "@type": "rdf:Property",
-            "rdfs:comment": "The short name of the instrument or item.",
-            "rdfs:label": "shortName",
-            "schema:domainIncludes": {
-                "@id": ""
-            },
+            "rdfs:comment": "An array to describe the order in which items appear in the ui.",
+            "rdfs:label": "order",
+            "schema:domainIncludes": [
+                {
+                    "@id": "reproschema:Protocol"
+                },
+                {
+                    "@id": "reproschema:Activity"
+                }
+            ],
             "schema:rangeIncludes": {
-                "@id": "schema:Text"
+                "@id": "schema:URL"
             },
             "schema:sameAs": {
-                "@id": "http://www.w3.org/2004/02/skos/core#altLabel"
+                "@id": "reproterms:order"
             }
         },
         {
-            "@id": "reproterms:ui",
+            "@id": "reproterms:shuffle",
             "@type": "rdf:Property",
-            "rdfs:comment": "An element to describe the ui of an Activity.",
-            "rdfs:label": "ui",
-            "schema:domainIncludes": {
-                "@id": "reproschema:Field"
+            "rdfs:comment": "An element (bool) to determine if the list of items is random or in order.",
+            "rdfs:label": "shuffle",
+            "schema:domainIncludes": [
+                {
+                    "@id": "reproschema:Protocol"
+                },
+                {
+                    "@id": "reproschema:Activity"
+                }
+            ],
+            "schema:rangeIncludes": {
+                "@id": "schema:boolean"
             },
+            "schema:sameAs": {
+                "@id": "reproterms:shuffle"
+            }
+        },
+        {
+            "@id": "reproschema:addProperties",
+            "@type": "rdf:Property",
+            "rdfs:comment": "An array of objects to describe the various properties added to assessments and fields.",
+            "rdfs:label": "addProperties",
+            "schema:domainIncludes": [
+                {
+                    "@id": "reproschema:Protocol"
+                },
+                {
+                    "@id": "reproschema:Activity"
+                }
+            ],
             "schema:rangeIncludes": {
-                "@id": "reproterms:order"
+                "@id": "schema:Text"
```

Comment:
@satra what should be range for `addProperties`?

Reply:
it should be a new object with those additional properties.